CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## v0.14.0
+* Update zq to [v0.18.0](https://github.com/brimsec/zq/releases/tag/v0.18.0)
+* Add [geolocation](https://github.com/brimsec/brim/wiki/Geolocation) data to Zeek `conn` logs generated from imported pcaps (#959, #957, #935)
+* Add developer documentation for [adding migrations](https://github.com/brimsec/brim/wiki/Adding-Migrations) (#921)
```

Comment:
Could you call these 'internal state migrations' or something, to reinforce that this is not a user concern?

Reply:
Sure thing. Went with your suggestion.